libtcmu.c

```diff
@@ -317,7 +319,9 @@ static struct nl_sock *setup_netlink(struct tcmulib_context *ctx)
 	 * Could be a older kernel. Ignore failure and just work in degraded
 	 * mode.
 	 */
-	set_genl_features(sock);
+	ret = set_genl_features(sock);
+	if (ret < 0)
```

Comment:
same here.

Reply:
It has record "tcmu_err("Could not set features. Error %d\n", ret);" before set_genl_features return.